vector/v.net.components/v.net.components.html

```diff
@@ -47,8 +47,3 @@ <h2>AUTHORS</h2>
 Daniel Bundala, Google Summer of Code 2009, Student<br>
 Wolf Bergenheim, Mentor<br>
 Markus Metz
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 7 missing one empty line.

Reply:
Line 43 `<a href="v.category.html">v.category</a>,` should be `<a href="v.category.html">v.category</a>`.